src/execute.rkt

```diff
@@ -0,0 +1,71 @@
+#lang racket
+
+(require "print/css.rkt")
+(provide execute-script script->js)
+
+
+(define (execute-script lines)
+  (define environment (make-hash))
+  (define effects '())
+  (for ([line (in-list lines)])
+    (match line
+      [`(let ,var ,expr)
+       (define val (evaluate-expr expr environment))
+       (hash-set! environment var val)]
+      [`(set ,var ,field ,expr)
+       (void)]
+      [`(append-child ,elt ,child)
+       (define elt* (evaluate-expr elt environment))
+       (define child* (evaluate-expr child environment))
+       (define box (elt->box child*))
+       (set! effects (cons (list 'append-child elt* child* box) effects))]))
+  (reverse effects))
+
+(define (evaluate-expr expr env)
+  (match expr
+    [(? symbol?)
+     (hash-ref env expr)]
+    [`(select ,sel)
+     sel]
+    [`(create ,tree)
+     tree]))
+
+(define (elt->box tree)
+  '[BLOCK])
+
+(define (script->js lines name)
+  (define js-lines '())
+  (set! js-lines (cons (format "function ~a(e) {" name) js-lines))
+  (for ([line (in-list lines)])
+    (define str
+      (match line
+        [`(let ,var ,expr)
+         (format "var ~a = ~a;" var (expr->js expr))]
+        [`(set ,var ,field ,expr)
+         (format "~a.~a = ~a;" var (expr->js field) (expr->js expr))]
```

Comment:
I wouldn't pass `field` through `expr->js`, since that isn't actually an expression. You could have a `field->js` function, I guess, or just put the match right here.